packages/web/stores/imperator-queries/pools.ts

```diff
@@ -0,0 +1,104 @@
+import { KVStore } from "@keplr-wallet/common";
+import {
+  ObservableQuery,
+  QueryOptions,
+  QueryResponse,
+} from "@keplr-wallet/stores";
+import Axios, { AxiosInstance, CancelToken } from "axios";
+import { makeObservable, observable, action, computed } from "mobx";
+import { computedFn } from "mobx-utils";
+import { PricePretty, CoinPretty, Dec, IntPretty } from "@keplr-wallet/unit";
+
+import { ObservablePool } from "@osmosis-labs/stores";
+
+import { IMPERATOR_API_DOMAIN } from "../../constants";
+import { FiatCurrency } from "@keplr-wallet/types";
+
+interface PoolMetric {
```

Comment:
I think you could de clutter and remove this interface and just put this type right on the `data:` field in the `ObservableImperatorQuery` base class type parameter.